sigs.yaml

```
@@ -2643,6 +2643,28 @@ workinggroups:
   contact:
     slack: wg-multitenancy
     mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-multitenancy
+- dir: wg-naming
+  name: Naming
```

Would something like `Inclusive Naming` make more sense and capture the intent of the WG? :thinking:

---

+1 If you just call this naming you'll get a bunch people asking regular naming questions.

`Inclusive Language` might capture the intent better, since I don't actually think you'll want to confine yourself to names / nouns.

---

+1 for `Inclusive Language`

---

Commenting this for lazy merging: 

```suggestion
  name: Inclusive Naming
```

---

One thing to note re: our original choice of WG **Naming** rather than WG Inclusive Naming, or WG D&I, etc.

I think that "diversity and inclusiveness" initiatives often start with great enthusiasm and then lose momentum quickly as people realize that the work requires deep, systemic change. This is a part of the reason for my [initial objection](https://github.com/kubernetes/kubernetes/issues/90277#issuecomment-641454365) to just making these changes in documentation – it's _easy_ to change the docs, but hard to enact systemic change on the codebase level. Similarly, it's _easy_ to launch a D&I committee, but hard to incorporate those practices in a meaningful, ongoing way to one's daily work.

I chose WG Naming because I wanted to emphasize that as much as this is about inclusiveness, it is, more importantly, about making systemic changes in the way that _everyone _works and thinks about these problems, rather than sidelining these changes to a passionate few, chartering a committee with little meaningful power to affect change. We're after inclusiveness, but the best green or fair trade products don't brand themselves that way – they deliver a great product _that happens to be_ fair trade.

We're a naming WG, that happens to seek inclusiveness.

---

+1 to "We're a naming WG, that happens to seek inclusiveness."

---

Agreed on naming.